package.json

```diff
@@ -24,7 +24,7 @@
   ],
   "scripts": {
     "test": "bundlesize && jest --coverage --no-cache",
-    "build": "npm run bundle && npm run minify",
+    "build": "npm run bundle && npm run minify && cp hyperapp.js.flow dist/",
```

Comment:
Not sure, would have to check with other libraries first, but I think:

```
   "files": [
     "src",
     "dist",
     "hyperapp.js.flow"
   ],
```

should work too no? @andyrj